packages/botonic-dx-bundler-rspack/baseline/rspack.config.ts

```diff
@@ -360,6 +362,30 @@ function botonicServerConfig(mode: string): Configuration {
   }
 }
 
+function botonicBotConfig(mode: Mode): Configuration {
+  return {
+    optimization: {
+      minimize: false,
+    },
+    context: ROOT_PATH,
+    devtool: false,
+    target: 'node',
+    entry: path.resolve(WEBPACK_ENTRIES_DIRNAME, WEBPACK_ENTRIES.BOT_CONFIG),
```

Comment:
Change: Since we're using rspack and not webpack, the dirname should be changed from `webpack-entries` to `rspack-entries`.